packages/sdk-js/src/GrowthBook.ts

```diff
@@ -129,10 +147,35 @@ export class GrowthBook<
   }
 
   public getApiInfo(): [ApiHost, ClientKey] {
-    return [
-      (this._ctx.apiHost || "https://cdn.growthbook.io").replace(/\/*$/, ""),
-      this._ctx.clientKey || "",
-    ];
+    return [this.getApiHosts().apiHost, this.getClientKey()];
+  }
+  public getApiHosts(): {
+    apiHost: string;
+    streamingHost: string;
+    apiRequestHeaders?: Record<string, string>;
+    streamingRequestHeaders?: Record<string, string>;
+    remoteEvalRequestHeaders?: Record<string, string>;
```

Comment:
These last 2 fields are never returned.  Do we still need them in the function signature?  Is this just to leave open the option of monkey patching?